ts/Series/Line/LineSeries.ts

```diff
@@ -179,9 +180,22 @@ class LineSeries extends Series {
                 }
 
                 graph[verb](attribs)
-                    // Add shadow to normal series (0) or to first
-                    // zone (1) #3932
-                    .shadow((i < 2) && options.shadow);
+                // Add shadow to normal series (0) or to first
+                // zone (1) #3932
+                    .shadow(
+                        (i < 2) &&
+                        (options.shadow ?
+                            // If shadow is defined, call function with
+                            // `filterUnits: 'userSpaceOnUse'` to avoid known
+                            // SVG filter bug (#19093)
+                            (isObject(options.shadow) ?
+                                merge(
+                                    options.shadow,
+                                    { filterUnits: 'userSpaceOnUse' }
+                                ) : { filterUnits: 'userSpaceOnUse' }
+                            ) : options.shadow
+                        )
+                    );
```

Comment:
I'm wondering if this can be shortened. Didn't test it, but something like this?
```js
.shadow(
    (i < 2) &&
    options.shadow &&
    merge(
        { filterUnits: 'userSpaceOnUse' },
        isObject(options.shadow) ? options.shadow : {}
    )
);
```

Reply:
Indeed, can be shortened; tested in local environment and works well.